maistra/scripts/update-envoy-sha.sh

```diff
@@ -47,7 +47,7 @@ function main() {
   get_envoy_sha
   get_envoy_sha_256
 
-  sed -i "s|^# Commit time: .*|# Commit time: ${today}|" WORKSPACE
+  sed -i "s|^# Commit date: .*|# Commit date: ${today}|" WORKSPACE
```

Comment:
If we are touching this, I'd suggest we 1) Get the correct commit date (it might not be today) and 2) Stick with the upstream format YYYY-MM-DD.

As a reference, this is the upstream (Istio) script: https://github.com/istio/proxy/blob/master/scripts/update_envoy.sh

Reply:
I'll approve, we can fix this in a follow up